test/functional/autoupdate/aup-basic.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	if [ -z "$TRAVIS" ]; then
+		skip "This test is intended to run only in Travis (use TRAVIS=true to run it anyway)..."
+	fi
+
+	create_test_environment "$TEST_NAME"
+
+}
+
+test_teardown() {
+
+	if [ -z "$TRAVIS" ]; then
+		return
+	fi
+
+}
+
+@test "AUT001: Basic test, Check auto-update enable" {
+
+	# perform autoupdate disable
+	run sudo sh -c "$SWUPD autoupdate --disable --path=$PATH_PREFIX"
+	assert_status_is "$SWUPD_OK"
+	expected_output=$(cat <<-EOM
+		Warning: disabling automatic updates may take you out of compliance with your IT policy
+		Running systemctl to disable updates
+	EOM
+	)
+	assert_is_output "$expected_output"
+
+	# check for files(should exist)
+	assert_file_exists "'$PATH_PREFIX'/etc/systemd/system/swupd-update.service"
```

Comment:
why the single quote?

Reply:
I thought I  needed it .. doesnt look like. Thanks.

Reply:
Done